frontend/src/metabase/setup/components/DatabaseStep/DatabaseStep.tsx

```diff
@@ -127,25 +122,37 @@ const DatabaseForm = ({
       formName="database"
       database={database}
       onSubmit={handleSubmit}
+      submitTitle={t`Connect database`}
     >
       {({
         Form,
         FormField,
-        FormFooter,
+        FormMessage,
+        FormSubmit,
         formFields,
         values,
         onChangeField,
+        submitTitle,
+        error,
       }: FormProps) => (
         <Form>
           <FormField name="engine" onChange={handleEngineChange} />
           <DriverWarning
             engine={values.engine}
             onChange={engine => onChangeField("engine", engine)}
           />
+          <FormError className="mt3 mb4" anchorMarginTop={24} error={error} />
           {_.reject(formFields, { name: "engine" }).map(({ name }) => (
             <FormField key={name} name={name} />
           ))}
-          {engine && <FormFooter submitTitle={t`Next`} />}
+          {engine && (
+            <>
+              <div className="flex mb2 justify-end">
+                <Button type="button" onClick={onSkip}>{t`Skip`}</Button>
+                <FormSubmit className="ml2">{submitTitle}</FormSubmit>
+              </div>
+            </>
+          )}
```

Comment:
Change markup to match the design
![image](https://user-images.githubusercontent.com/1937582/169349022-c43f9fed-3f73-4f84-b93a-b673f92a36f7.png)